### ISSUE TITLE
After 1.1.0 upgrade: Failed to initialize ORM engine: strconv.ParseInt: parsing "...": invalid syntax

### ISSUE THREAD

**Issue by Funzinator** [go-gitea/gitea#1267]
- Gitea version (or commit ref): 1.1.0
- Git version: 2.11.1
- Operating system: FreeBSD
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:
```
2017/03/15 17:01:36 [I] Log Mode: File(Info)
2017/03/15 17:01:36 [I] XORM Log Mode: File(Info)
2017/03/15 17:01:36 [I] Cache Service Enabled
2017/03/15 17:01:36 [I] Session Service Enabled
2017/03/15 17:01:36 [I] Mail Service Enabled
2017/03/15 17:01:36 [I] Notify Mail Service Enabled
2017/03/15 17:01:36 [I] Migration: create repo unit table and add units for all repos
2017/03/15 17:01:37 [I] Migration: set protect branches updated with created
2017/03/15 17:01:37 [I] Migration: add external login user
2017/03/15 17:01:37 [I] Migration: generate and migrate Git hooks
2017/03/15 17:01:38 [I] Migration: use new avatar path name for security reason
2017/03/15 17:01:38 [...itea/routers/init.go:54 GlobalInit()] [E] Failed to initialize ORM engine: migrate: do migrate: strconv.ParseInt: parsing "b3f11dbac06e95476f532b503cf70d8c": invalid syntax
```
## Description
After I had the same issue like #1261 and dropped the 2 indexes, I got this error. I have no idea where b3f11dbac06e95476f532b503cf70d8c comes from.

**Comment by xoxys** [comment]
Same here

**Comment by Funzinator** [comment]
short update: The hash (b3f11...) comes from an avatar that has been renamed from id to the hash. But the upgrade failed after moving the avatar files and retries to rename all avatar files (already hashed, not a valid int value) to hashes on each start.

Hope that helps.

For now I reverted the upgrade using a previous backup and didn't try again yet.

**Comment by strk** [comment]
@lunny this one seems for you, please give highest priority. We'll need a better way to test upgrades before releases (call community for test would be great)

**Comment by lunny** [comment]
We have fixed this via #1197. Please confirm that PR resolved your problem or wait for v1.1.1

**Comment by lunny** [comment]
It should be fixed by #1197.
